app/views/coronavirus_pages/_sub_sections_summary_list.html.erb

```diff
@@ -1,8 +1,9 @@
-
+<% reorder ||= false %>
```

Comment:
I think there should be a separate partial for the reorder display, rather than reusing this partial.